src/index.html

```diff
@@ -3,12 +3,55 @@
   <head>
     <meta charset="UTF-8">
     <meta name="viewport"
-          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header>
+      <a
+        href="#home"
+        class="logo"
+      >
+        <img
+          src="./images/logo.png"
+          alt="Logo"
+        >
+      </a>
+      <nav>
+        <ul>
+          <li>
+            <a href="#" class="is-active">apple</a>
+          </li>
+          <li>
+            <a href="#">samsung</a>
+          </li>
+          <li>
+            <a href="#">smartphones</a>
+          </li>
```

Comment:
```suggestion
          <li>
            <a href="#">samsung</a>
          </li>
          
          <li>
            <a href="#">smartphones</a>
          </li>
```
add blanks between children. it's better for readability.